.github/workflows/check-new-links.yml

```diff
@@ -0,0 +1,40 @@
+name: Check new links
```

Comment:
Sometimes I get PR review (Pavlo) and have to check URLS manually. Sometimes they don't work as like RN is not ready but its ok I think. Are you planning to make this Required?

Reply:
> Are you planning to make this Required?

No.